### ISSUE TITLE
Implementation docs

### ISSUE THREAD

**Issue by 9SL9** [skiselkov/librain#1]
Hi 

Your project looks very interesting. Please do let us know if you have any documentation available. 

https://forums.x-plane.org/index.php?/forums/topic/161811-librain-rain-library-for-windshield-effects/

Cheers

**Comment by nico87** [comment]
+1. Docs are really needed to use the library :-)

**Comment by henrikx** [comment]
maybe make a quickstart guide from the ground up including example implementation?

**Comment by skiselkov** [comment]
Working on it right now! I should have it ready by the end of the week.

**Comment by parshukov** [comment]
> 
> 
> Working on it right now! I should have it ready by the end of the week.

Good news! We are waiting)))

**Comment by nico87** [comment]
Any news? :-)

**Comment by kbrandwijk** [comment]
+1 on the implementation docs @skiselkov. I love to include it in another airplane, but I have no idea where to start...

**Comment by isaacjairi** [comment]
+1 on the implementation docs 
https://github.com/skiselkov
i'm trying but no success....

**Comment by henrikx** [comment]
PR #3 looks pretty good

**Comment by skiselkov** [comment]
Yep, will integrate it soon. Just working on wiper support right now. Then I'll integrate the docs in the PR.
